qa/tasks/cephfs/test_client_recovery.py

```diff
@@ -395,14 +395,21 @@ def test_filelock_eviction(self):
         self.assertFalse(lock_holder.finished)
         self.assertFalse(lock_taker.finished)
 
-        mount_a_client_id = self.mount_a.get_global_id()
-        self.fs.mds_asok(['session', 'evict', "%s" % mount_a_client_id])
+        try:
+            mount_a_client_id = self.mount_a.get_global_id()
+            self.fs.mds_asok(['session', 'evict', "%s" % mount_a_client_id])
 
-        # Evicting mount_a should let mount_b's attepmt to take the lock
-        # suceed
-        self.wait_until_true(
-            lambda: lock_taker.finished,
-            timeout=10)
+            # Evicting mount_a should let mount_b's attepmt to take the lock
+            # suceed
```

Comment:
L403: comment typo 'succeed' ?

Reply:
NIT: wrong spelling of 'attepmt'.

Even why this comment is highlighted as changed?